### ISSUE TITLE
Desktop copy and paste

### ISSUE THREAD

**Issue by wtruax** [klinker-apps/messenger-issues#627]
- [ ] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my issue/request to verify that there aren't any open or closed issues around this request.
- [ ] If this is an issue, and not a request, I have verified that this is something that is reproducible

**Device with issue:** <insert your device: "Samsung Laptop running Windows 10 home">
**Primary device:** <insert your phone: "Samsung Glaxy S7 edge on Android 7.0">
**App version:** <insert version: 2.9.2.1974>
**Steps to reproduce:**

1. Right Click any text on desktop

**Additional information:**

There are no right-click options for the text message such as forward, copy text, etc. The mobile app has long press options to copy, share, delete, info. On desktop, I can't highlight the text or manipulate the information.

**Possible solution:**

**Comment by klinker24** [comment]
The only one missing is the forward functionality, or am I missing something? The web app has a compose button at the top of the screen, so it is already much more available than the Android one.

**Comment by wtruax** [comment]
Yes, forward or share. Either one. thank you for the quick reply.

**Comment by klinker24** [comment]
I don't think share will come, it doesn't really make sense on the desktop, all it would do is copy the text for you. I will see about adding a forward button in the future.

Forwarding is so easy already though, right now. Just drag to select the text and hit the "+" button at the top of the screen. This hasn't come to the web version of the app, while it is on Android, because it is so much simpler to start a new conversation on the web. It just isn't all that necessary
